lib/__tests__/navigation.test.js

```diff
@@ -17,123 +17,121 @@ describe('Navigation metalsmith plugin', () => {
+      'section1/test-pattern/with-additional-segment/index.html': {
         contents: Buffer.from('test pattern with additional directory segment'),
         mode: '0644',
         stats: {},
-        filepath: 'test/test-pattern/with-additional-segment/index.njk'
+        filepath: 'section1/test-pattern/with-additional-segment/index.njk'
       },
-      'test/development-test-pattern/index.html': {
+      'section1/development-test-pattern/index.html': {
         contents: Buffer.from('test pattern with development status'),
         mode: '0644',
         stats: {},
-        filepath: 'test/development-test-pattern/index.njk',
+        filepath: 'section1/development-test-pattern/index.njk',
         status: 'development'
       },
-      'test/secondary-navigation-item/index.html': {
+      'section1/secondary-navigation-item/index.html': {
         contents: Buffer.from('test pattern in secondary navigation position'),
         mode: '0644',
         stats: {},
-        filepath: 'test/secondary-navigation-item/index.njk',
+        filepath: 'section1/secondary-navigation-item/index.njk',
         navigationPlacement: 'secondary'
       },
-      'test/hidden-item/index.html': {
+      'section1/hidden-item/index.html': {
         contents: Buffer.from('test pattern hidden from menu'),
         mode: '0644',
         stats: {},
-        filepath: 'test/hidden-item/index.njk',
+        filepath: 'section1/hidden-item/index.njk',
         navigationPlacement: 'none'
       }
     }
     const expected = {
-      navigation: [
-        {
-          filepath: 'test/test-pattern-with-custom-text/index.html',
-          href: 'test/test-pattern-with-custom-text',
-          placement: 'primary',
-          section: 'test',
-          text: 'Custom text!'
-        },
-        {
-          filepath: 'test/secondary-navigation-item/index.html',
-          href: 'test/secondary-navigation-item',
-          placement: 'secondary',
-          section: 'test',
-          text: 'Secondary navigation item'
-        },
-        {
-          filepath: 'test/test-pattern/index.html',
-          href: 'test/test-pattern',
-          placement: 'primary',
-          section: 'test',
-          text: 'Test pattern'
-        },
-        {
-          filepath: 'test/test-pattern-with-more-words/index.html',
-          href: 'test/test-pattern-with-more-words',
-          placement: 'primary',
-          section: 'test',
-          text: 'Test pattern with more words'
-        },
-        {
-          filepath: 'another/testing/index.html',
-          href: 'another/testing',
-          placement: 'primary',
-          section: 'another',
-          text: 'Testing'
-        },
-        {
-          filepath: 'test/aardvark/index.html',
-          href: 'test/aardvark',
-          placement: 'primary',
-          section: 'test',
-          text: 'Zebra'
-        }
-      ]
+      navigation: {
+        'section1': [
```

Comment:
Would it be helpful if the section can have more than just a string? The format would be:

```
{
  navigation: [
    {
      title: 'section1',
      items: [ ... ]
    }
  ]
}
```